### PR TITLE
Delete symlinks on Cloudready before install

### PR DESCRIPTION
**Pull request by supechicken**
On Cloudready, `var` is a symlink which pointed to `/var` by default
```shell
lrwxrwxrwx  1 root    root        1 Nov 10 11:49 var -> /var
```

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -33,7 +33,12 @@ case "${ARCH}" in
 esac
 
 # This will allow things to work without sudo
-sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
+sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}/*/*"
+# Delete 'var' symlink on Cloudready platform
+if [[ $(cat /etc/lsb-release | grep neverware) != "" ]]; then
```
Comment (uberhacker) [inline install.sh:38]: Easier logic?
```
if [[ $(grep -c neverware /etc/lsb-release) > 0 ]]; then
```

File: install.sh [hunk 2]
```diff
@@ -33,7 +33,12 @@ case "${ARCH}" in
 esac
 
 # This will allow things to work without sudo
-sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
+sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}/*/*"
```
Comment (uberhacker) [inline install.sh:36]: Why do we need to change this line?

Comment (supechicken) [reply install.sh]: Removed

File: install.sh (hunk 1 above)
Comment (supechicken) [reply install.sh]: Done

**Comment by uberhacker** [comment]
I'm not sure of the ramifications of doing this but on the surface it looks correct.

**Pull request merged**